lib/Markup/CMakeLists.txt

```diff
@@ -3,8 +3,4 @@ add_swift_host_library(swiftMarkup STATIC
   LineList.cpp
   Markup.cpp)
 target_link_libraries(swiftMarkup PRIVATE
-  libcmark-gfm_static)
-target_compile_definitions(swiftMarkup
-                           PRIVATE
-                           CMARK_GFM_STATIC_DEFINE)
-
+  libcmark-gfm)
```

Comment:
To make sure I understand correctly, https://github.com/apple/swift-cmark/pull/69 makes `libcmark-gfm` a `STATIC` library by default, and only Windows builds turns `BUILD_SHARED_LIBS=ON` when building Cmark, right?

Reply:
Correct, nothing changes for the other platforms.